entries/off.xml

```diff
@@ -26,6 +26,12 @@
   </signature>
   <signature>
     <added>1.7</added>
+    <argument name="event" type="Event">
+      <desc>A <a href="/category/events/event-object/"><code>jQuery.Event</code></a> object.</desc>
+    </argument>
+  </signature>
+  <signature>
+    <added>1.7</added>
```

Comment:
This signature was around earlier than 1.7, I had to keep it working when I refactored the code in 1.7. I'll look for the right version.